packages/server/lib/modes/record.js

```diff
@@ -108,145 +108,315 @@ const getSpecRelativePath = (spec) => {
   return _.get(spec, 'relative', null)
 }
 
-const uploadArtifacts = (options = {}) => {
-  const { protocolManager, video, screenshots, videoUploadUrl, captureUploadUrl, screenshotUploadUrls, quiet } = options
+const humanReadableFilesize = (fileSize) => {
+  const KB = 1000
```

Comment:
Should these be 1024 not 1000?

Reply:
I assumed s3 limits were 5 GB rather than 5 GiB, so used base 10

Reply:
Should we show filesizes in base 10 (kB, MB, etc) or base 2 (kiB, MiB)?